big/2/reconnect.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Run all instances
+cd atomic_disc_drive
+
+cargo build
+[ $? -ne 0 ] && exit 1
+
+tmp1=$(mktemp -d)
+tmp2=$(mktemp -d)
+tmp3=$(mktemp -d)
+
+cargo run ../simple_config 1 ${tmp1} & 
+cargo run ../simple_config 2 ${tmp2} &
+cargo run ../simple_config 3 ${tmp3} &
+cd ../
+
+sleep 4
+
+# Run simple write test
+cd ./public-tests/
+cargo test --test reconnect external_write -- --ignored
```

Comment:
I suggest checking exit code to possibly end test on failure.
Proposed solution requires reading test logs to see whether it passed.

Reply:
Ex. use the same as below: `[ $? -ne 0 ] && echo "FAIL"`